lib/Sema/TypeCheckDecl.cpp

```diff
@@ -4027,10 +4027,7 @@ ParamSpecifierRequest::evaluate(Evaluator &evaluator,
   return ParamSpecifier::Default;
 }
 
-static void validateParameterType(ParamDecl *decl) {
-  if (auto ty = decl->getTypeLoc().getType())
-    return;
-
+static Type validateParameterType(ParamDecl *decl) {
```

Comment:
*Gee, this whole thing looks suspiciously requestifiable now*

Reply:
DOOOOO EEETTTT